pkg/controller/constrainttemplate/constrainttemplate_controller.go

```diff
@@ -63,8 +65,9 @@ const (
 )
 
 var (
-	logger       = log.Log.V(logging.DebugLevel).WithName("controller").WithValues("kind", "ConstraintTemplate", logging.Process, "constraint_template_controller")
-	discoveryErr *apiutil.ErrResourceDiscoveryFailed
+	logger         = log.Log.V(logging.DebugLevel).WithName("controller").WithValues("kind", "ConstraintTemplate", logging.Process, "constraint_template_controller")
+	discoveryErr   *apiutil.ErrResourceDiscoveryFailed
+	vapEnforcement = flag.Bool("vap-enforcement", false, "control VAP resource generation. Allowed values are false: do not generate unless generateVAP: true is added to policy explicitly, true: generate unless generateVAP: false is added to policy explicitly.")
```

Comment:
pls update this as well

Reply:
As a beta feature, we want to enable the flag to `true` by default, right?

Reply:
done